app/services/tweet_service.rb

```diff
@@ -0,0 +1,15 @@
+class TweetService
+  def self.conn
+    Faraday.new(url: 'https://api.twitter.com') do |faraday|
+      faraday.headers['authorization'] = ENV['twitter_token']
```

Comment:
This line needs to be:
faraday.headers['Authorization'] = "Bearer #{ENV['twitter_token']}"
to authorize with twitter

Reply:
That makes so much more sense that what I had @brueck1988 do. TY! Alexander, make sure you reflect the change on your local `application.yml` but let me know if you're not sure how 😃